src/blocks/checkout/customer.tsx

```
@@ -1,17 +1,283 @@
+  const [taxIdType, setTaxIdType] = useState("CPF");
+  const onSubmit = async (data: CustomerInformation) => {
+    try {
+      setCustomerInfo(data);
+      setCurrentStep("payment");
+      await router.push("/checkout?step=payment");
+    } catch (error) {
+      console.error("Failed to process customer information:", error);
+    }
+  };
+
+  const onError = (errors: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)